dags/top-public-synapse-projects-from-snowflake.py

```diff
@@ -28,15 +21,24 @@
 from airflow.providers.snowflake.hooks.snowflake import SnowflakeHook
 from orca.services.synapse import SynapseHook
 from slack_sdk import WebClient
+import json
 
 dag_params = {
-    "snowflake_developer_service_conn": Param("SNOWFLAKE_DEVELOPER_SERVICE_RAW_CONN", type="string"),
+    "snowflake_developer_service_conn": Param(
+        "SNOWFLAKE_DEVELOPER_SERVICE_RAW_CONN", type="string"),
     "synapse_conn_id": Param("SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN", type="string"),
-    # hours_time_delta is the number of hours to subtract from the current date to get the date for the query
+    # hours_time_delta is the number of hours to subtract from the current date to get
+    # the date for the query
     "hours_time_delta": Param("24", type="string"),
     "backfill": Param(False, type="boolean"),
     # backfill_date string format: YYYY-MM-DD
     "backfill_date": Param("1900-01-01", type="string"),
+    # fileview_groups is a JSON string containing an array of objects with file_view_id
+    # and group_name properties
+    # Example: '[{"file_view_id": "123456", "group_name": "Group A"},
+    #           {"file_view_id": "789012", "group_name": "Group B"}]'
+    "fileview_groups": Param(
```

Comment:
This might be a time to use the `array` or `object` param type
https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/params.html#use-params-to-provide-a-trigger-ui-form